community-stabilization-fund-api/src/services/form-response/index.tsx

```diff
@@ -1,25 +1,32 @@
-import getConfig from "next/config";
+import getConfig from 'next/config';
 
-import { axiosInstance } from "../constants";
+import { axiosInstance } from '../constants';
 
-import type { FormResponse } from "../../db";
+import type { FormResponse } from '../../db';
 
 const { publicRuntimeConfig } = getConfig() || {};
 const baseUrl = `${publicRuntimeConfig?.apiUrl}/form-responses`;
+const formResResetUrl = `${publicRuntimeConfig?.apiUrl}/form-responses/reset-form-responses`;
 
 const FormResponseService = {
   getAllFormResponses,
   createFormResponse,
+  resetFormResponse,
 };
 
 async function getAllFormResponses() {
   const res = await axiosInstance.get<FormResponse[]>(`${baseUrl}`);
   return res;
-};
+}
 
 async function createFormResponse() {
   const res = await axiosInstance.post<FormResponse>(`${baseUrl}`);
   return res;
-};
+}
+
+async function resetFormResponse() {
+  const res = await axiosInstance.post<FormResponse>(`${formResResetUrl}`);
```

Comment:
use the baseUrl and just append the rest of the url like `${baseUrl}/bulk-delete`